src/api/app/models/staging_workflow.rb

```diff
@@ -1,17 +1,18 @@
 class StagingWorkflow < ApplicationRecord
   belongs_to :project, inverse_of: :staging
-  has_many :staging_projects, class_name: 'Project', inverse_of: :staging_workflow, dependent: :nullify, autosave: true do
+  has_many :staging_projects, class_name: 'Project', inverse_of: :staging_workflow, dependent: :nullify do
```

Comment:
Just as reminder, with merging of https://github.com/openSUSE/open-build-service/pull/6193 we can drop all changes in this file.